### ISSUE TITLE
Regex interpolation does not work

### ISSUE THREAD

**Issue by dlee** [opal/opal#509]
Opal:

``` ruby
>> a = /a/
=> /a/
>> /#{a}/
=> //a//
>> /#{a}/ =~ 'a'
=> nil
```

Ruby:

``` ruby
>> a = /a/
=> /a/
>> /#{a}/
=> /(?-mix:a)/
>> /#{a}/ =~ 'a'
=> 0
```

**Comment by ysakasin** [comment]
This bug seems to be fixed on Opal 1.0.0, right?

**Comment by elia** [comment]
@ysakasin that's correct! thanks for the heads up
